src/mgr/BasicActions.cc

```diff
@@ -50,17 +51,24 @@ Mgr::MenuAction::MenuAction(const Command::Pointer &aCmd): Action(aCmd)
 void
 Mgr::MenuAction::dump(StoreEntry* entry)
 {
-    debugs(16, 5, MYNAME);
     Must(entry != nullptr);
+    PackableStream os(*entry);
+
+    const auto &menu = CacheManager::GetInstance()->menu();
 
-    typedef CacheManager::Menu::const_iterator Iterator;
-    const CacheManager::Menu& menu = CacheManager::GetInstance()->menu();
+    const auto savedFlags = os.flags();
```

Comment:
There is no need to save and restore flags here AFAICT because this code does not "export" the stream object (i.e. `os`) to some other code after restoring the flags.

```suggestion
```

Reply:
Sorry, forgot to note that this PR depends on PR 1560.  Changes in this method are copied from there to enable build.